client/landing/gutenboarding/components/plans/plans-table/style.scss

```diff
@@ -1,6 +1,7 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 @import '../../../mixins.scss';
 @import '../../../variables.scss';
+@import 'assets/stylesheets/shared/mixins/placeholder'; // Contains the placeholder mixin
```

Comment:
We might have this in Gutenboarding separately since we do something like this for domains when searching.

Reply:
We use the same one https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/components/domain-picker/style.scss#L2